sdk/python/kfp/v2/google/client/client.py

```diff
@@ -368,6 +368,7 @@ def create_schedule_from_job_spec(
       service_account: Optional[str] = None,
       enable_caching: Optional[bool] = None,
       app_engine_region: Optional[str] = None,
+      service_account_for_schedule: Optional[str] = None,
```

Comment:
Can we update the name to `cloud_scheduler_service_account` to better reflect the name? Otherwise LGTM, thanks!

Reply:
@ji-yaqi Thanks for the feedback! Renamed in https://github.com/kubeflow/pipelines/pull/6013/commits/77c18d71e4b240e36d252e47b0952e9934226690 👍